utils/build_swift/tests/expected_options.py

```diff
@@ -0,0 +1,494 @@
+    StrOption('--darwin-xcrun-toolchain', dest='darwin_xcrun_toolchain'),
+    StrOption('--enable-tsan-runtime', dest='enable_tsan_runtime'),
+    StrOption('--host-target', dest='host_target'),
+    StrOption('--lit-args', dest='lit_args'),
+    StrOption('--llvm-targets-to-build', dest='llvm_targets_to_build'),
+
+    PathOption('--android-deploy-device-path',
+               dest='android_deploy_device_path'),
+    PathOption('--android-icu-i18n', dest='android_icu_i18n'),
+    PathOption('--android-icu-i18n-include', dest='android_icu_i18n_include'),
+    PathOption('--android-icu-uc', dest='android_icu_uc'),
+    PathOption('--android-icu-uc-include', dest='android_icu_uc_include'),
+    PathOption('--android-ndk', dest='android_ndk'),
+    PathOption('--build-subdir', dest='build_subdir'),
+    PathOption('--clang-profile-instr-use', dest='clang_profile_instr_use'),
+    PathOption('--cmake', dest='cmake'),
+    PathOption('--coverage-db', dest='coverage_db'),
+    PathOption('--host-cc', dest='host_cc'),
+    PathOption('--host-cxx', dest='host_cxx'),
+    PathOption('--host-libtool', dest='host_libtool'),
+    PathOption('--host-lipo', dest='host_lipo'),
+    PathOption('--install-prefix', dest='install_prefix'),
+    PathOption('--install-symroot', dest='install_symroot'),
+    PathOption('--symbols-package', dest='symbols_package'),
+
+    IntOption('--benchmark-num-o-iterations',
+              dest='benchmark_num_o_iterations'),
+    IntOption('--benchmark-num-onone-iterations',
+              dest='benchmark_num_onone_iterations'),
+    IntOption('--jobs', dest='build_jobs'),
+    IntOption('--llvm-max-parallel-lto-link-jobs',
+              dest='llvm_max_parallel_lto_link_jobs'),
+    IntOption('--swift-tools-max-parallel-lto-link-jobs',
+              dest='swift_tools_max_parallel_lto_link_jobs'),
+    IntOption('-j', dest='build_jobs'),
+
+    AppendOption('--cross-compile-hosts', dest='cross_compile_hosts'),
+    AppendOption('--extra-cmake-options', dest='extra_cmake_options'),
+    AppendOption('--extra-swift-args', dest='extra_swift_args'),
+    AppendOption('--stdlib-deployment-targets',
+                 dest='stdlib_deployment_targets'),
+
+    UnsupportedOption('--build-jobs'),
+    UnsupportedOption('--common-cmake-options'),
+    UnsupportedOption('--ios-all'),
+    UnsupportedOption('--only-execute'),
+    UnsupportedOption('--skip-test-optimize-for-size'),
+    UnsupportedOption('--skip-test-optimized'),
+    UnsupportedOption('--tvos-all'),
+    UnsupportedOption('--watchos-all'),
+    UnsupportedOption('-I'),
+
+    # FIXME: LTO flag is a special case that acts both as an option and has
+    # valid choices
+    Option('--lto', dest='lto_type'),
+    ChoicesOption('--lto', dest='lto_type', choices=['thin', 'full']),
+
+    # NOTE: We'll need to manually test the behavior of these since they
+    # validate compiler version strings.
+    _BaseOption('--clang-compiler-version',
```

Comment:
Same here. Can you make a wrapper rather than using an _ class? Looks weird to me.

Reply:
Will do!